crates/app/ironclaw_cli/.skills/portfolio/widget/style.css

```
@@ -0,0 +1,121 @@
+.pf-card {
+  padding: 16px;
+  font-family: -apple-system, Segoe UI, Roboto, sans-serif;
```

Stylelint (17.14.0)

[error] Expected quotes around "Segoe UI" (font-family-name-quotes)

(font-family-name-quotes)